app/models/site.rb

```diff
@@ -203,4 +203,7 @@ def self.clear_anon_cache!
     MessageBus.publish(SITE_JSON_CHANNEL, '')
   end
 
+  def self.show_welcome_topic_banner?
```

Comment:
We should only showi the welcome topic banner to staff, or even to the first admin user. Would be good to add a spec for this as well.

Reply:
Done, thanks for the suggestion!